### PR TITLE
ping-pong

### PR DESCRIPTION
**Pull request by andreev-io**
… infer online status

### CODE REVIEW

File: Airtap/Network/WS/WSService.swift [hunk 1]
```diff
@@ -152,13 +152,12 @@ class WSService: WSServing {
     }
     
     private func handle(_ message: WSPayload) {
-        guard let accountId = message.payload?.fromAccountId else { return }
-        
         switch(message.type) {
         case .ack:
             self.logProvider.add(LogLevel.debug, String(format: "account %d received an ack", self.authProvider.accountId!))
```
Comment (alxdrlitreev) [inline Airtap/Network/WS/WSService.swift]: Please avoid force unwrap. Replace `self.authProvider.accountId!` with `self.authProvider.accountId ?? "UNKNOWN"`

File: Airtap/Network/WS/WSService.swift [hunk 2]
```diff
@@ -152,13 +152,12 @@ class WSService: WSServing {
...
             break
         case .offer:
+            guard let accountId = message.payload?.fromAccountId else { return }
             self.logProvider.add(LogLevel.debug, String(format: "account %d received an offer from %d", self.authProvider.accountId!, accountId))
```
Comment (alxdrlitreev) [inline Airtap/Network/WS/WSService.swift]: Same here.

File: Airtap/Network/WS/WSService.swift [hunk 3]
```diff
@@ -170,6 +169,7 @@ class WSService: WSServing {
                 )
             }
         case .answer:
+            guard let accountId = message.payload?.fromAccountId else { return }
             self.logProvider.add(LogLevel.debug, String(format: "account %d received an answer from %d", self.authProvider.accountId!, accountId))
```
Comment (alxdrlitreev) [inline Airtap/Network/WS/WSService.swift]: And here.

File: Airtap/Network/WS/WSService.swift [hunk 4]
```diff
@@ -181,6 +181,7 @@ class WSService: WSServing {
                 )
             }
         case .candidate:
+            guard let accountId = message.payload?.fromAccountId else { return }
             self.logProvider.add(LogLevel.debug, String(format: "account %d received a candidate from %d", self.authProvider.accountId!, accountId))
```
Comment (alxdrlitreev) [inline Airtap/Network/WS/WSService.swift]: Here.

File: Airtap/Network/WS/WSService.swift [hunk 5]
```diff
@@ -194,6 +195,7 @@ class WSService: WSServing {
                 )
             }
         case .info:
+            guard let accountId = message.payload?.fromAccountId else { return }
             self.logProvider.add(LogLevel.debug, String(format: "account %d received info from %d", self.authProvider.accountId!, accountId))
```
Comment (alxdrlitreev) [inline Airtap/Network/WS/WSService.swift]: And here.

File: Airtap/Network/WS/WSService.swift [hunk 6]
```diff
@@ -205,7 +207,10 @@ class WSService: WSServing {
                 )
             }
         case .peers:
-            self.logProvider.add(LogLevel.debug, String(format: "account %d received peers \(message.payload as Optional)", self.authProvider.accountId!))
+            self.sendAck(nonce: message.nonce)
+            if let peers = message.payload?.onlinePeers {
+                self.logProvider.add(LogLevel.debug, String(format: "account %d received peers \(message.payload as Optional)", self.authProvider.accountId!, peers))
```
Comment (alxdrlitreev) [inline Airtap/Network/WS/WSService.swift]: Aaand here.

Comment (alxdrlitreev) [reply Airtap/Network/WS/WSService.swift]: Check if there's somewhere else in the app we have it in such a way for logging.

File: Airtap/Network/WS/WSService.swift (hunk 1 above)
Comment (alxdrlitreev) [reply Airtap/Network/WS/WSService.swift]: Also, no need to write `LogLevel.debug`. Just do `.debug`. 
Apply it to everywhere else.

**Review by alxdrlitreev — CHANGES_REQUESTED** [review]

**Comment by andreev-io** [comment]
Fixed and addressed all.

**Pull request merged**